ecosystem/typescript/sdk/src/transaction_builder/builder_utils.ts

```diff
@@ -178,6 +185,9 @@ export class TypeTagParser {
       this.consume(">");
       return new TypeTagVector(res);
     }
+    if (tokenVal === "string" || tokenVal === "String") {
+      return new TypeTagStruct(stringStructTag);
+    }
```

Comment:
I would rather move the string => 0x1::string::String translation to `deserializeValueBasedOnTypeTag` and `getPropertyValueRaw`. It is safer to move it there.  `TypeTagParser` parses type tag string by adhereing to Move standard types. TypeTagParser is also being used as a reference by some community GoLang SDK. Adding a "string" translation here would have a broader impact.

Reply:
updated